advertising/hotel-feed/reference.md

```diff
@@ -61,7 +61,7 @@ Defines a hotel.
 |longitude|Required.<br /> Data type is decimal.<br /><br />The longitude of the hotel's geographical coordinates.<br /><br />Notes:<ul><li>The latitude and longitude are required only if you don't specify `phone`. It's preferred that you specify both the phone and geographical coordinates.</li><li>The longitude must be in the range -180.0 through 180.0.</li><li>Use [Location API](https://msdn.microsoft.com/library/ff701715.aspx) or another GeoCoding tool to generate the coordinates.</li></ul>|None
 |<a name="phone"></a>phone|Required.<br />Data type is string.<br /><br />A list of telephone numbers that customers use to contact the hotel.<br /><br />Attributes:<ul><li>type&mdash;Required. The type of telephone number specified. The following are the possible values.<br /><ul><li>main&mdash;Required. The hotel's main voice telephone number.</li><li>tollfree&mdash;Optional. The hotel's toll-free voice telephone number.</li><li>fax&mdash;Optional. The hotel's fax telephone number.</li><li>tdd&mdash;Optional. The hotel's telecommunications device for the deaf telephone number.</li><li>mobile&mdash;Optional. The hotel's mobile telephone number.</li></ul></ul>Notes:<ul><li>A phone number is required only if you don't specify `latitude` and `longitude`. It's preferred that you specify both the phone and geographical coordinates.</li><li>Provide unique telephone numbers for the listing.</li><li>Use dashes, spaces, or parentheses in the phone number to make it easier to read. For example, use “610-222-3333” or “(610) 222-3333” rather than “6102223333”.</li><li>Specify only one telephone number in each `phone` element. Do not specify multiple numbers, such as 650-123-2222/33.</li><li>The telephone number may contain an extension of up to 7 digits. Precede extensions with one of the following abbreviations: "ext", "extn", and "x". For example, "408-555-1111x12345" or "408-555-1111 x12345".</li><li>The telephone number may not include alphabetical characters.</li><li>Each listing should specify one of each type of phone number: "main", "mobile", "tollfree", "fax", or "tdd", if applicable.</li><li>If the telephone number includes the country code, precede it with a "+". For example, “+65 6722-2323” for a number in Singapore where the country code is 65, or “+001 (408) 555-1111” for a number in the United States where the country code is 001.</li></ul>|None
-|category|Optional.<br />Data type is string.<br /><br/>A user-defined category that identifies the type of hotel. For example, extended stay, economy, or motel.|None
+|category|Optional.<br />Data type is string.<br /><br/>A user-defined category that identifies the type of hotel. For example, extended stay, economy, or motel. Max character length: 200 |None
```

Comment:
@rixMS - Also is there a minimum that you can mention?
```suggestion
|category|Optional.<br />Data type is string.<br /><br/>A user-defined category that identifies the type of hotel. For example, extended stay, economy, or motel. The maximum character length is 200. |None
```

Reply:
@rixMS - Yes this can be blank, so in that sense there isn't any minimum. However, sometimes *if* values are provided (not empty), the service can enforce a minimum. Let's merge this PR and you can open a new PR to clarify any minimum as needed. Thanks!
@rgaritta FYI